packages/uix-host-react/src/hooks/useExtensions.ts

```diff
@@ -131,6 +136,17 @@ export function useExtensions<
     const newExtensions = [];
     const guests = host.getLoadedGuests(requires);
     for (const guest of guests) {
+      //proceed only if this guest supports provided extension point
+      if (
+        currentComponentExtensionPoint &&
+        guest.extensionPoints &&
```

Comment:
Please add unit/integration tests or a Storybook to ensure that the logic (for all cases) is working as expected. 

For example, what will happen if currentComponentExtensionPoint is not defined (extensionPoint was not specified), but guest.extensionPoints contains some values? In this case the condition will no be satisfyed. Is it desire state?

Reply:
I will add testing support in project (currently there is none), so it will take some more time.